nifi-extension-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/AbstractExecuteSQL.java

```diff
@@ -531,4 +554,38 @@ protected List<String> getQueries(final String value) {
     }
 
     protected abstract SqlWriter configureSqlWriter(ProcessSession session, ProcessContext context, FlowFile fileToProcess);
+
+    enum ContentOutputStrategy implements DescribedValue {
+        EMPTY_RESULT(
+            "Overwrite content",
```

Comment:
"Title Case" should be used for these property values.
```suggestion
            "Overwrite Content",
```